panels/user-accounts/cc-user-panel.ui

```diff
@@ -438,6 +438,48 @@
                         <property name="height">1</property>
                       </packing>
                     </child>
+                    <child>
+                      <object class="GtkLabel" id="parental_controls_label">
+                        <property name="visible">True</property>
+                        <property name="xalign">1</property>
+                        <property name="label" translatable="yes">Parental Controls Settings</property>
+                        <property name="use_underline">True</property>
+                        <property name="mnemonic_widget">malcontent_launch_button</property>
+                        <style>
+                          <class name="dim-label"/>
+                        </style>
+                      </object>
+                      <packing>
+                        <property name="left_attach">0</property>
+                        <property name="top_attach">7</property>
+                        <property name="width">1</property>
+                        <property name="height">1</property>
+                      </packing>
+                    </child>
+                    <child>
+                      <object class="GtkButton" id="malcontent_launch_button">
+                        <property name="visible">True</property>
+                        <property name="can_focus">True</property>
+                        <property name="receives_default">True</property>
+                        <signal name="clicked" handler="launch_malcontent_control" object="CcUserPanel" swapped="yes"/>
+                        <style>
+                          <class name="text-button"/>
+                        </style>
+                        <child>
+                          <object class="GtkLabel" id="malcontent_launch_button_label">
+                            <property name="label" translatable="yes">Launch</property>
+                            <property name="visible">True</property>
+                            <property name="halign">GTK_ALIGN_START</property>
+                          </object>
+                        </child>
+                      </object>
+                      <packing>
+                        <property name="left_attach">1</property>
+                        <property name="top_attach">7</property>
+                        <property name="width">1</property>
+                        <property name="height">1</property>
```

Comment:
Maybe add `<property name="x-options"></property>` to set the button so it doesn’t expand or fill? It looks a bit weird that wide.

Reply:
I agree with this but probably also need to take a look at the other similar buttons as well for e.g. `Last login` and `Password:`

![Screenshot from 2020-03-04 16-38-45](https://user-images.githubusercontent.com/8775041/75873883-bcfa8100-5e36-11ea-9d74-95e33f1d4569.png)

I will not address this here but probably will try to include the change in the 3.36-rebase branch itself.
Thanks for pointing it out.

Reply:
FYI,

the new UI for user-accounts panel looks something like this:
![Screenshot from 2020-03-04 16-51-15](https://user-images.githubusercontent.com/8775041/75874824-8b82b500-5e38-11ea-82c4-321f83a55862.png)


So will adapt this particular patch to it.

Reply:
There’s a task for adapting this patch to 3.36 [here](https://phabricator.endlessm.com/T29440), and it includes links to the upstream designs for exactly this button :)